src/renderer/src/pass/deferred.rs

```diff
@@ -32,16 +32,21 @@ impl<R> ::Pass<R> for Clear
 pub static DRAW_VERTEX_SRC: &'static [u8] = b"
     #version 150 core
 
-    uniform mat4 u_Proj;
-    uniform mat4 u_View;
-    uniform mat4 u_Model;
+    layout (std140) uniform u_VertexArgs {
```

Comment:
Nothing wrong with this, but we need to discuss the naming of stuff at some point. With DX11 on board, it kinda went off the original plan. For the constant buffers, we may consider `cb_Something` scheme.